components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -95,15 +96,18 @@ class CustomTabsToolbarFeature(
     }
 
     @VisibleForTesting
-    internal fun updateToolbarColor(@ColorInt toolbarColor: Int?) {
+    internal fun updateToolbarColor(@ColorInt toolbarColor: Int?, @ColorInt navigationBarColor: Int?) {
         toolbarColor?.let { color ->
             toolbar.setBackgroundColor(color)
             toolbar.textColor = readableColor
             toolbar.titleColor = readableColor
             toolbar.siteSecurityColor = Pair(readableColor, readableColor)
             toolbar.menuViewColor = readableColor
 
-            window?.setStatusBarTheme(toolbarColor)
+            window?.setStatusBarTheme(color)
+        }
+        navigationBarColor?.let { color ->
+            window?.setNavigationBarTheme(color)
```

Comment:
What do you think about trying to fallback to trying with the toolbar color since that might be used more than the the navigation bar colour?

Reply:
I think its better to be a no-op in this case, since the default navigation bar is the expected behaviour.